packages/admin/admin/src/common/Tooltip.tsx

```diff
@@ -14,9 +14,11 @@ import { cloneElement, ComponentProps, useState } from "react";
 import { createComponentSlot } from "../helpers/createComponentSlot";
 
 export interface TooltipProps extends MuiTooltipProps {
+    // @deprecated Triggers other than the default "hover" will be removed in the future.
```

Comment:
Does a line comment work correctly for deprecations? Shouldn't this be JSDoc? Is it shown in the IDE?

Reply:
Aah good catch, no that didn't work, it does now 🙂